source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst

```diff
@@ -97,8 +97,8 @@ Refer :ref:`Release Notes <Release-note-label>` for complete feature-set and fur
 .. note::
    Processor SDK Linux AM62A contains only the Linux specific source and application intended
```

Comment:
For all directives under this file, **Make sure that the note directive and the content block have 1 new line between them and that the content block has at least 1 newline after it.**

Refer CONTRIBUTING guidelines for more details - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace